blog-posts/boosting-developer-productivity-with-github-actions.md

```diff
@@ -0,0 +1,266 @@
+
+![A comment by github-actions that points to the corresponding release](./images/github-actions/release-comment.jpg)
+
+#### "Housekeeping"
+Housekeeping is an important aspect of the fast-paced frontend world. We used to have an issue to remind us to update our dependencies every once in a while. As with a lot of manual processes we forgot to do this from time to time. Especially when it comes to important vulnerability fixes it’s important to keep up to date. We now have a workflow to take care of that once a week using a scheduled event. With integrated tools like "Dependabot" this workflow might be obsolete soon, but so far we enjoy the fact that we are in full control of the workflow and can tweak and configure it to our hearts content.
+
+## Driven by laziness
+Most of these examples are driven by laziness or the very human forgetfulness. It's the laziness to not assign a certain project to a new issue or the laziness to not run `yarn upgrade` once in a while. Basically the point of this blog post is to inspire you with the inherent laziness that I discovered in myself over the past few years. 
+
+## It's not about the code
+All these examples have one more thing in common: They are not necessarily related to the code in the repository. They rather deal with processes and metadata. They automate the things that we have to deal with on a daily basis that have to do with issue management, release cycles, vulnerabilities in third-party dependencies and so on.
+
+Other external automation platforms like Jenkins, TravisCI or CircleCI are certainly splendid tools when it comes to automation, but from my experience, nothing integrates as smoothly into GitHub as "GitHub Actions" do. This is not particularly surprising, given the name - but it's still worth mentioning.
+
+While external automation platforms sure provide a lot of value - and will continue to do so - there are aspects of a repository that are easiest automated using GitHub Actions (given that it lives inside GitHub).
+The convenience with which you can interact with Projects, Issues, Releases and other non-code related features of GitHub have been a crucial deciding factor for me, when choosing _where_ to automate things. I won't advocate migrating all your pipelines to GitHub Actions. Instead I suggest to start small. Start with the things that would be hard to automate with your existing automation infrastructure and then slowly migrate whatever feels right.
+
+## Think out of the box!
+> "A repository is more than the sum of its code. Automate the repository, not the code."
+
+There is a lot more to automation then the typical "continuous integration" and "continuous delivery" pipelines. Think outside of the box of CI/CD and be creative.
```

Comment:
In addition and unrelated to the former comment about the last sections of the blog post:

I know you are not a fan of conclusion or story telling but this blog post leaves me very abruptly. No good bye, not final words, not good luck doing the same not wrapping up the stuff you told me (which was a lot). 

I think a conclusion "I explained the terminology and then showed you the awesomeness of github actions" would make a really nice wrap up for a really good blog post. Such a conclusion helps people to store the knowledge much better as you basically summarize the "red thread" (is this even a saying in English?).

Reply:
This time I completely agree. I've slightly shuffled the ending sections around and added a more "conclusion" like ending. Thanks for suggesting this 👌

Reply:
I saw then, thank you. I appreciate that a lot! I will put my new comments in there and resolve this.